### PR TITLE
Add duration of last job execution to the table

### PR DESCRIPTION
**Pull request by nickvergessen**
Downstream of 27144

### CODE REVIEW

File: db_structure.xml [hunk 1]
```diff
@@ -1012,6 +1012,14 @@
 				<notnull>false</notnull>
 			</field>
 
+			<field>
+				<!-- time for execution of the job -->
+				<name>execution_duration</name>
+				<type>integer</type>
+				<default></default>
+				<notnull>true</notnull>
```
Comment (ChristophWurst) [inline db_structure.xml]: I hope all DBMSs handle adding new colums with `notnull=true` correctly if the table has data in it. If the default value is applied it *should* be fine of course … \*fingers crossed\*

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: db_structure.xml (hunk 1 above)
Comment (MorrisJobke) [inline db_structure.xml]: It works here with MySQL just fine.

Comment (MorrisJobke) [inline db_structure.xml]: @icewind1991 Could you test on postgres?

**Review by ChristophWurst — APPROVED** [review]
Aside from https://github.com/nextcloud/server/pull/4501/files#r113265911 it looks good!

File: db_structure.xml (hunk 1 above)
Comment (MorrisJobke) [inline db_structure.xml]: I tested and it works fine on Postgres as well 👍

**Pull request merged**